packages/Confirmation/stories/screener.stories.js

```diff
@@ -1,11 +0,0 @@
-import React from "react";
-import { storiesOf } from "@storybook/react";
-import { Story } from "storybook/assets/styles/common.styles";
-
-import ConfirmationExample from "./examples/ConfirmationExample";
-
-storiesOf("Confirmation / screener", module).add("confirmation test", () => (
```

Comment:
This is not the only example where I've consolidated `stories.js` files, there were a few places it seemed simpler to do so, but here especially.